content/docs/command-reference/params/index.md

```diff
@@ -143,6 +143,98 @@ $ dvc run -n train -d logs/ -o users.csv \
           python train.py
 ```
 
+## Examples: Python parameters file
+
+Consider this parameters file in Python format, named `params.py`:
+
+```python
+IS_BOOL: bool = True
+CONST = 5
+
+# All standard variable types are supported
+FLOAT = 0.001
+STR = 'abc'
+DICT = {
+    "a": 1,
+    "b": 2
+}
+LIST = [1, 2, 3]
+SET = {4, 5, 6}
+TUPLE = (10, 100)
+NONE = None
+
+
+# It is possible to retrieve either class constants
+# or own variables defined in __init__
+class TrainConfig:
+    EPOCHS = 70
+
+    def __init__(self):
+        # TrainConfig.layers param will be 9
+        self.layers = 5
+        self.layers = 9
+        # TrainConfig.foo will NOT be found because the complex expression
+        self.foo = 1 + 2
+        # TrainConfig.bar will NOT be found
+        bar = 1
+
+
+class TestConfig:
+    TEST_DIR = "path"
+    METRICS = ["metric"]
+```
+
+The following [stage](/doc/command-reference/run) depends on params `IS_BOOL`,
+`CONST`, as well as `TrainConfig`'s `EPOCHS` and `layers`:
+
+```dvc
+$ dvc run -n train -d users.csv -o model.pkl \
+          -p params.py:IS_BOOL,CONST,TrainConfig.EPOCHS,TrainConfig.layers \
+          python train.py
+```
+
+Resulting `dvc.yaml` and `dvc.lock` files (notice the `params` list):
+
+```yaml
+stages:
+  train:
+    cmd: python train.py
+    deps:
+      - users.csv
+    params:
+      - IS_BOOL
+      - CONST
+      - TrainConfig.EPOCHS
+      - TrainConfig.layers
+    outs:
+      - model.pkl
+```
+
+```yaml
+train:
+  cmd: python train.py
+  deps:
+    - path: users.csv
+      md5: 23be4307b23dcd740763d5fc67993f11
+  params:
+    CONST: 5
+    IS_BOOL: true
+    TrainConfig.EPOCHS: 70
+    TrainConfig.layers: 9
+  outs:
+    - path: model.pkl
+      md5: 1c06b4756f08203cc496e4061b1e7d67
+```
+
+Alternatively, the entire `TestConfig` group can be referenced (also a
+dictionary), instead of the parameters in it:
```

Comment:
Per https://github.com/iterative/dvc.org/pull/1799#pullrequestreview-497944318:
```suggestion
Alternatively, the entire `TestConfig` group (class) can be referenced
(dictionaries are also supported), instead of the parameters in it:
```
Maybe link class to https://docs.python.org/3.8/library/stdtypes.html#classes-and-class-instances
and dictionary to https://docs.python.org/3.8/library/stdtypes.html#dict

Reply:
I agree